sdk/storage/ci.yml

```diff
@@ -1,55 +1,55 @@
 # NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
-trigger:
-  branches:
-    include:
-      - master
-      - feature/*
-      - release/*
-      - hotfix/*
-  paths:
-    include:
-      - cmake-modules/
-      - eng/
-      - CMakeLists.txt
-      - sdk/core
-      - sdk/identity
-      - sdk/storage
+# trigger:
+#   branches:
+#     include:
+#       - master
+#       - feature/*
+#       - release/*
+#       - hotfix/*
+#   paths:
+#     include:
+#       - cmake-modules/
+#       - eng/
+#       - CMakeLists.txt
+#       - sdk/core
+#       - sdk/identity
+#       - sdk/storage
 
-pr:
-  branches:
-    include:
-      - master
-      - feature/*
-      - release/*
-      - hotfix/*
-  paths:
-    include:
-      - cmake-modules/
-      - eng/
-      - CMakeLists.txt
-      - sdk/core/
-      - sdk/identity/
-      - sdk/storage
+# pr:
+#   branches:
+#     include:
+#       - master
+#       - feature/*
+#       - release/*
+#       - hotfix/*
+#   paths:
+#     include:
+#       - cmake-modules/
+#       - eng/
+#       - CMakeLists.txt
+#       - sdk/core/
+#       - sdk/identity/
+#       - sdk/storage
 
-stages:
-  - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
-    parameters:
-      ServiceDirectory: storage
-      # Not running tests on CI but only on LiveTest mode
-      CtestRegex: no-run
-      Coverage: disabled
-      LiveTestCtestRegex: azure-storage
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-      Artifacts:
-        - Name: azure-storage-common
-          Path: azure-storage-common
-          VcpkgPortName: azure-storage-common-cpp
-        - Name: azure-storage-blobs
-          Path: azure-storage-blobs
-          VcpkgPortName: azure-storage-blobs-cpp
-        - Name: azure-storage-files-datalake
-          Path: azure-storage-files-datalake
-          VcpkgPortName: azure-storage-files-datalake-cpp
-        - Name: azure-storage-files-shares
-          Path: azure-storage-files-shares
-          VcpkgPortName: azure-storage-files-shares-cpp
+# stages:
+#   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+#     parameters:
+#       ServiceDirectory: storage
+#       # Not running tests on CI but only on LiveTest mode
+#       CtestRegex: no-run
+#       Coverage: disabled
+#       LiveTestCtestRegex: azure-storage
+#       SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+#       Artifacts:
+#         - Name: azure-storage-common
+#           Path: azure-storage-common
+#           VcpkgPortName: azure-storage-common-cpp
```

Comment:
Intentional change?

Reply:
Yes, storage CI is disabled until it is fixed